docs/schemas/resource/IfcPresentationAppearanceResource/Entities/IfcIndexedPolygonalTextureMap.md

```diff
@@ -0,0 +1,16 @@
+# IfcIndexedPolygonalTextureMap
+
+The _IfcIndexedPolygonalTextureMap_ provides the mapping of the 2-dimensional texture coordinates to a set of polygonal bounded faces onto which it is mapped. It is used for mapping the texture to faces of an _IfcPolygonalFaceSet_. Such faces may have inner loops.
+
+The _TexCoords_ defined at supertype _IfcIndexedTextureMap_ are a two-dimensional list of texture coordinates provided by two parameter values. Each member of the set of _TexCoordIndices_ defines an unbounded index list corresponding to the vertices of the assigned polygonal face that points to the texture coordinates in _TexCoords_ for that vertex.
```

Comment:
> two-dimensional list ... provided by two parameter 

This might confuse the reader as to whether it's 2d list of 2 parameters or whether the 2 parameters are already included in the rank. I propose:

"two-dimensional list of texture coordinates providing by two parameter values for every vertex."